tools/GeneratePythonLibrary.sh

```diff
@@ -126,9 +126,10 @@ function run-tests {
   fi
 
   cd tests
+  rm -f kaggle kagglesdk
```

Comment:
Is this intentional?

Reply:
Yes. Sometimes, when there's an error in the script, it doesn't clean up the symlinks. This just prevents a little frustration.